src/server/status/__tests__/cgroup.js

```diff
@@ -177,7 +176,22 @@ describe('Control Group', function () {
       });
       const stats = await getAllStats();
 
-      expect(stats).to.be.null;
+      expect(stats).toEqual({
```

Comment:
Oups. This one didn't actually do an assertion before 🙈 Not sure if this is the correct result though. I just added an assertion for the actual value of `stats`

Reply:
@tylersmalley Thoughts on this one?

Reply:
Looking into it

Reply:
This update is correct 👍 

If the cpu.stat is missing we continue to provide the rest of the stats.